main.go

```diff
@@ -24,7 +24,7 @@ func main() {
 	db.SetupRedisPool()
 	oauthServerErr := oauth.SetupOAuthServer()
 	if oauthServerErr != nil {
-		log.Fatal(fmt.Sprintf(startServerFailureError, oauthServerErr))
+		log.Fatal(fmt.Sprintf("Failed to start oauth server: %v", oauthServerErr))
```

Comment:
How about adding another constant to be consistent?

Reply:
Fixed in 3046d67caea8ccc5ddcdcc103ee60462136f5492